examples/Tester/Tests/Server/UnitTests/Helpers/TestServerStreamWriter.cs

```diff
@@ -77,6 +79,10 @@ public Task WriteAsync(T message)
 
             return Task.CompletedTask;
         }
+
+        public Task WriteAsync(T message)
```

Comment:
Why not just provide a default value above?

Reply:
Can't because these are interface methods.